### PR TITLE
Docs/style guide and diataxis review3

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description

This PR aims to update the docs structure and text style to follow the [TWs guide](https://wiki.iota.org/participate/contribute-to-wiki/for_tws/writer_guide) and the [diataxis framework](https://diataxis.fr/)

**Cherry-picked from https://github.com/gohornet/hornet/pull/1330 , but targetting the mainet branch now.**

## Type of change

- [ ] Documentation Enchancement

# How Has This Been Tested?

Wiki was built locally.

# Checklist:

- [ ] I have selected the `mainnet` branch as the target branch

### CODE REVIEW

File: documentation/docs/getting_started/security_101.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 description: This section provides a checklist of steps for running a reliable and secure node.
 image: /img/logo/HornetLogo.png
 keywords:
+description: This section provides a checklist of steps for running a reliable and secure node.
```
Comment (muXxer) [inline documentation/docs/getting_started/security_101.md]: That looks wrong to me, why is it here twice?

File: documentation/docs/welcome.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 ---
 
 # Welcome to HORNET
 
 ![Hornet Logo](/img/logo/HORNET_logo.svg)
 
-HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end
-devices like the Raspberry Pi 4. a group of community developers alongside the
-IOTA Foundation are responsible for building and maintaining Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the
+HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end devices like the Raspberry Pi 4., a group of community developers alongside the IOTA Foundation are responsible for building and maintaining Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the
```
Comment (muXxer) [inline documentation/docs/welcome.md]: We should remove the Raspi 4 from all the docs.

File: documentation/docs/getting_started/security_101.md (hunk 1 above)
Comment (lucas-tortora) [reply documentation/docs/getting_started/security_101.md]:
```suggestion
```

File: documentation/docs/welcome.md (hunk 2 above)
Comment (lucas-tortora) [reply documentation/docs/welcome.md]: I think my last commit should take care of that.  Please review the hornet_apt_repository.md file

File: documentation/docs/welcome.md [hunk 3]
```diff
@@ -2,39 +2,44 @@
...
 # Welcome to HORNET
 
 ![Hornet Logo](/img/logo/HORNET_logo.svg)
 
-HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end devices like the Raspberry Pi 4., a group of community developers alongside the IOTA Foundation are responsible for building and maintaining Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the
-Chrysalis network update.
+HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end
+devices, a group of community developers alongside the IOTA Foundation are responsible for building and maintaining
+Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the Chrysalis
+network update.
```
Comment (muXxer) [inline documentation/docs/welcome.md]:
```suggestion
HORNET is a powerful, easy to install IOTA node software written in Go.
It provides the full node capabilities including full support of the Chrysalis network update.
```

File: documentation/docs/getting_started/security_101.md [hunk 4]
```diff
@@ -2,7 +2,6 @@
 description: This section provides a checklist of steps for running a reliable and secure node.
 image: /img/logo/HornetLogo.png
 keywords:
-description: This section provides a checklist of steps for running a reliable and secure node.
 image: /img/logo/HornetLogo.png
```
Comment (muXxer) [inline documentation/docs/getting_started/security_101.md]: keywords and image is there twice as well.

File: documentation/docs/getting_started/hornet_apt_repository.md [hunk 5]
```diff
@@ -104,7 +104,6 @@ All installation methods mentioned in this article from this point should be con
    Please make sure to download the binaries for your system:
    
    * `HORNET-X.Y.Z_Linux_x86_64.tar.gz`: standard 64-bit-linux-based executable, such as Ubuntu, Debian, etc.
-   * `HORNET-X.Y.Z_Linux_arm64.tar.gz`: executable for Raspberry Pi 4.
```
Comment (muXxer) [inline documentation/docs/getting_started/hornet_apt_repository.md:107]: I wouldn't remove it, but rewrite it to `executable for 64bit ARM based systems.`

**Review by muXxer — APPROVED** [review]

**Pull request merged**